src/db/index.js

```diff
@@ -0,0 +1,3 @@
+const Tasks = [{TaskId:1,TaskName:"Singing",TaskPriority:"high"}];
```

Comment:
What is the reason for this data in this array?

Reply:
I wanted to test my endpoint